electronics/ohms_law.py

```diff
@@ -0,0 +1,32 @@
+# Importing doctest for testing our function
+from doctest import testmod
```

Comment:
Please move this import inside `__main__` so I want to call your function from a bigger program, I do not need to import doctest.

Also, please replace the comment with something that helps the reader understand your algorithm like https://en.wikipedia.org/wiki/Ohm%27s_law